src/components/like-button/button.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { LikeButton } from './like-button';
+
+const meta = {
+  title: 'Component/LikeButton',
+  component: LikeButton,
+  parameters: {
+    layout: 'centered',
+  },
+  tags: ['autodocs'],
+  argTypes: {
+    onClick: { action: 'clicked' },
+  },
+} satisfies Meta<typeof LikeButton>;
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Unliked: Story = {
+  args: {},
+};
+
+export const Liked: Story = {
+  args: {
+    isLikedByUser: true,
+  },
+};
+
+export const Like: Story = {
+  args: {
+    likes: 1,
+  },
+};
```

Comment:
ich würde 1 story daraus machen:

```ts
export const LikedByUser: Story = {
  args: {
    isLikedByUser: true,
    likes: 1
  },
};

Reply:
Ich habe einfach Liked replaced, in Figma wenn liked by user, gibts kein counter